src/client/index.ts

```diff
@@ -338,6 +339,16 @@ class Client extends EventEmitter {
     return transactionUtils.convertStringToHex(string);
   }
 
+  /**
+   * Autofills missing fields in a transaction
+   *
+   * @param {Transaction} tx A transaction to autofill missing fields
+   * @returns {Transaction} An autofilled transaction
+   */
+  async autofillTransaction(tx: Transaction): Promise<Transaction> {
+    return transactionUtils.autofillTransaction(tx, this)
+  }
```

Comment:
why not call this `autofill`? the signature will make it clear that we expect a transaction, and there aren't other things to autofill. Also, shouldn't you `await` the function from `transactionUtils`? Otherwise you could instead write this as:

```typescript
import { partial } from 'lodash'

autofillTransaction = partial(transactionUtils.autofillTransaction, _, this)
```

Reply:
I agree with your suggestions. However, I'm not a fan of importing a module when this can be done natively with JavaScript's `bind()`:
```typescript
autofill = transactionUtils.autofill.bind(null, this);
```
What do you think?

Reply:
i'm no javascript expert, but in my opinion the weirdness of `this` in javascript is such that it should be avoided. but if you'd like to chat about this more i'm down! maybe this is the right move.

Reply:
@ledhed2222 and I further discussed this in a meeting and we agreed to create a sugar function for JavaScript `bind()`. These changes have been included.